helm/minio/templates/post-job.yaml

```diff
@@ -78,6 +78,9 @@ spec:
               - key: {{ .Values.tls.publicCrt }}
                 path: CAs/public.crt
         {{- end }}
+      {{ if .Values.serviceAccount.create }}
```

Comment:
Would you mind changing to the following to delete extra new line?
```yaml
{{- if .Values.serviceAccount.create }}
```

Reply:
Good catch, fixed :)